coconut/configuration/configuration.go

```diff
@@ -390,17 +387,15 @@ func Import(cfg *configuration.ConsulSource, cmd *cobra.Command, args []string,
 
 	userMsg := ""
 	if !componentExist {
-		userMsg = "A new component has been created: " + red(component) + "\n"
-	} else {
-		userMsg += "Component " + red(component) + " has been updated" +  "\n"
+		userMsg = "New component created: " + red(component) + "\n"
 	}
 	if !entryExists {
-		userMsg += "A new entry has been created: " + blue(entry) + "\n"
+		userMsg += "New entry created: " + blue(entry) + "\n"
 	} else {
-		userMsg += "Entry " + blue(entry) + " has been updated" +  "\n"
+		userMsg += "Entry updated: " + blue(entry) +  "\n"
 	}
 	fullKey :=  red(component) + "/" + blue(entry) + "@" + strconv.FormatInt(timestamp, 10)
-	userMsg += "The following configuration key has been added: " + fullKey
+	userMsg += "Configuration created: " + fullKey
```

Comment:
I know we discussed this at length trying to find the best wording and we settled on this, but after additional thought I realized that:
1) this string is the only one that's always printed (the ones for new entry and component aren't)
2) this string is the last (and sometimes only) line of output in response to a `coconut conf import` command.
---
Therefore I propose the following:
`userMsg += "Configuration imported: " + fullKey`